editor/editor_run.cpp

```diff
@@ -68,9 +68,18 @@ Error EditorRun::run(const String &p_scene, const String p_custom_args, const Li
 
 	int screen = EditorSettings::get_singleton()->get("run/window_placement/screen");
 	if (screen == 0) {
+		// Same as editor
 		screen = OS::get_singleton()->get_current_screen();
+	} else if (screen == 1) {
+		// Leftwards monitor (wrap to the other end if needed)
+		screen = abs((OS::get_singleton()->get_current_screen() - 1) % OS::get_singleton()->get_screen_count());
```

Comment:
You should use `Math::abs`, `abs` is not portable.

Reply:
Also, are we sure that all OSes identify monitors from left to right?

Reply:
> Also, are we sure that all OSes identify monitors from left to right?

I don't know, it seems to be the case on Linux at least.